instabot/api/api_photo.py

```diff
@@ -246,3 +246,70 @@ def resize_image(fname):
     new.paste(img, (0, 0, w, h), img)
     new.save(new_fname, quality=95)
     return new_fname
+
+
+def stories_shaper(fname):
+    '''
+    Find out the size of the uploaded image.
+    Processing is not needed if the image is already 1080x1920 pixels.
+    Otherwise, the image height should be 1920 pixels.
+    Substrate formation: Crop the image under 1080x1920 pixels and apply a Gaussian Blur filter.
+    Centering the image depending on its aspect ratio and paste it onto the substrate.
+    Save the image.
+    '''
+    try:
+        from PIL import Image, ImageFilter
+    except ImportError as e:
+        print("ERROR: {}".format(e))
+        print("Required module `PIL` not installed\n"
+              "Install with `pip install Pillow` and retry")
+        return False
+    img = Image.open(fname)
+    if (img.size[0], img.size[1]) == (1080, 1920):
+        print("Image is already 1080x1920. Just converting image.")
+        new_fname = "{}.STORIES.jpg".format(fname)
+        new = Image.new("RGB", (img.size[0], img.size[1]), (255, 255, 255))
+        new.paste(img, (0, 0, img.size[0], img.size[1]))
+        new.save(new_fname)
+        return new_fname
+    else:
+        min_width = 1080
```

Comment:
looks like these are the constants. I'd suggest making them global variables at the beginning of the file. And replace all `1080` and `1920`.